runelite-client/src/main/java/net/runelite/client/plugins/grounditems/ItemThreshold.java

```diff
@@ -94,4 +94,18 @@ boolean quantityHolds(int itemCount)
 			return itemCount > quantity;
 		}
 	}
+
+	public static boolean matchesEntry(String entry, String itemName)
+	{
+		int index = entry.indexOf('<');
+		if (index == -1)
+		{
+			index = entry.indexOf('>');
+		}
+		if (index != -1)
+		{
+			entry = entry.substring(0, index);
```

Comment:
Hi! Is there ever a chance that the variable entry ever contains trailing whitespaces? If so, one should add .trim() to row 107 as otherwise equalsIgnoreCase without trimming will lead to an incorrect result

Reply:
No https://github.com/runelite/runelite/blob/f8420ab53559574662546ca8595831f2bb2d8786/runelite-client/src/main/java/net/runelite/client/util/Text.java#L60